mycli/packages/parseutils.py

```diff
@@ -64,6 +64,9 @@ def last_word(text, include='alphanum_underscore'):
 # This code is borrowed from sqlparse example script.
 # <url>
 def is_subselect(parsed):
+    if isinstance(parsed, sqlparse.sql.Token):
+        if not parsed.is_group:
+            return False
     if not parsed.is_group():
```

Comment:
I'm not sure; I need to test it. If it's changed to is_group without () the tests fail... So somehow the unit tests are hitting this code.